src/disco/fd_disco_base.h

```diff
@@ -87,6 +90,18 @@ FD_FN_CONST static inline ulong fd_disco_poh_sig_pkt_type( ulong sig ) { return
 FD_FN_CONST static inline ulong fd_disco_poh_sig_slot( ulong sig ) { return (sig >> 8); }
 FD_FN_CONST static inline ulong fd_disco_poh_sig_bank_tile( ulong sig ) { return (sig >> 2) & 0x3FUL; }
 
+FD_FN_CONST static inline ulong
+fd_disco_replay_sig( ulong slot,
+                     ulong flags ) {
+   /* The high 7 bits of the low byte of the signature field is the bank
```

Comment:
Comment seems wrong, and do you need functions to extract the slot and flags as well?

Reply:
That will get added with the replay tile changes in a follow up